core/api/service/state/state_api.hpp

```diff
@@ -19,6 +22,9 @@ namespace kagome::api {
         const common::Buffer &key) const = 0;
     virtual outcome::result<common::Buffer> getStorage(
         const common::Buffer &key, const primitives::BlockHash &at) const = 0;
+    /// Returns Version of the Runtime.
```

Comment:
Just to be consistent, please add comments to all methods. And elaborate a bit on them, because in my opinion it is no more informative than the method name.